packages/odie-client/src/constants.tsx

```diff
@@ -243,6 +243,33 @@ export const getOdieInitialMessage = (
 	botNameSlug: OdieAllowedBots,
 	displayName: string
 ): Message => {
+	if ( botNameSlug === 'automattic-chat-support_a4a' ) {
+		return {
+			content: `**${ sprintf(
+				/* translators: %(name)s: the user's display name */
+				__( 'Howdy %(name)s 👋', __i18n_text_domain__ ),
+				{
+					name: displayName || 'there',
+				}
+			) }** \n\n ${ __(
+				"I'll get your chat started and then pass you to a human. What do you need help with today?",
+				__i18n_text_domain__
+			) }`,
+			role: 'bot',
+			type: 'introduction',
+			context: getOdieInitialPromptContext( botNameSlug ),
+			metadata: {
+				quick_replies: [
+					__( 'WordPress.com', __i18n_text_domain__ ),
+					__( 'WooCommerce', __i18n_text_domain__ ),
+					__( 'Jetpack', __i18n_text_domain__ ),
+					__( 'Automattic for Agencies', __i18n_text_domain__ ),
+					__( 'Pressable', __i18n_text_domain__ ),
```

Comment:
I have a hunch that we'd want an untranslated ID for these. Something like 

```suggestion
					{ id: 'wordpress.com', text: __( 'WordPress.com', __i18n_text_domain__ ) } ,,
```

Better than handing fuzzy text when we want to decide around the choice (for Pressable).

Or maybe add on more prop called "navigateTo" to these and call `navigate( value )` when it is set and the item is clicked? This way you can go to the contact form in case of Pressable.

Reply:
Good point! Yes, ID makes a lot of sense here.